deployability/modules/jobflow/tests/conftest.py

```diff
@@ -19,7 +19,7 @@
 @pytest.fixture
 def logger_mock(request) -> MagicMock:
     """Fixture to mock common logger methods."""
-    logger_to_patch = request.param.get('logger_to_patch', "workflow_engine.workflow_processor.logger")
+    logger_to_patch = request.param.get('logger_to_patch', "jobflow.workflow_processor.logger")
```

Comment:
Shouldn't the name of the workflow_processor method be changed?

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8